src/current/v24.1/cockroach-start.md

```diff
@@ -100,10 +100,12 @@ Flag | Description
 `--enterprise-encryption` | This optional flag specifies the encryption options for one of the stores on the node. If multiple stores exist, the flag must be specified for each store. <br /><br /> This flag takes a number of options.  For a complete list of options, and usage instructions, see [Encryption at Rest]({% link {{ page.version.version }}/encryption.md %}). <br /><br /> Note that this is an [Enterprise feature]({% link {{ page.version.version }}/enterprise-licensing.md %}).
 <a name="flags-external-io-disable-http"></a>`--external-io-disable-http` |  This optional flag disables external HTTP(S) access (as well as custom HTTP(S) endpoints) when performing bulk operations (e.g, [`BACKUP`]({% link {{ page.version.version }}/backup.md %}), [`IMPORT`]({% link {{ page.version.version }}/import.md %}), etc.). This can be used in environments where you cannot run a full proxy server. <br><br>If you want to run a proxy server, you can start CockroachDB while specifying the `HTTP(S)_PROXY` environment variable.
 `--external-io-disable-implicit-credentials` |  This optional flag disables the use of implicit credentials when accessing external cloud storage services for bulk operations (e.g, [`BACKUP`]({% link {{ page.version.version }}/backup.md %}), [`IMPORT`]({% link {{ page.version.version }}/import.md %}), etc.).
+`--node-cert-distinguished-name` <a name="flags-node-cert-distinguished-name"></a> | A string with a comma separated list of distinguished name (DN) mappings in `{attribute-type}={attribute-value}` format in accordance with [RFC4514](https://www.rfc-editor.org/rfc/rfc4514) for the [`node` user]({% link {{ page.version.version }}/security-reference/authorization.md %}#node-user). If this flag is set, this needs to be an exact match with the DN subject in the client certificate provided for the `node` user. For more information, see [Certificate-based authentication using multiple values from the X.509 Subject field]({% link {{ page.version.version }}/certificate-based-authentication-using-the-x509-subject-field.md %}).
+`--root-cert-distinguished-name` <a name="flags-root-cert-distinguished-name"></a> | A string with a comma separated list of distinguished name (DN) mappings in `{attribute-type}={attribute-value}` format in accordance with [RFC4514](https://www.rfc-editor.org/rfc/rfc4514) for the [`root` user]({% link {{ page.version.version }}/security-reference/authorization.md %}#root-user). If this flag is set, this needs to be an exact match with the DN subject in the client certificate provided for the `root` user. For more information, see [Certificate-based authentication using multiple values from the X.509 Subject field]({% link {{ page.version.version }}/certificate-based-authentication-using-the-x509-subject-field.md %}).
```

Comment:
By "exact match" does it have to be in the same order?

Reply:
ditto re: hopefully will be answered in other convo

Reply:
section 2 of https://www.rfc-editor.org/rfc/rfc4514 mentions `DistinguishedName ::= RDNSequence`, which implies that attributes used in distinguished name follow a particular sequence and needed to be provided in that order, not all attributes need to be present, but if only a few are present, they still need to maintain order.

Reply:
thanks @souravcrl! 

I will update this section of the doc as well to add the info that exact match also means ordering must be maintained

Reply:
OK, updated this text to add another sentence saying that the order of attributes must match